app/models/manageiq/providers/autosde/inventory/parser/storage_manager.rb

```diff
@@ -151,6 +152,20 @@ def cloud_volumes
     end
   end
 
+  def cloud_volume_snapshots
+    collector.cloud_volume_snapshots.each do |snapshot|
+      cloud_volume = persister.cloud_volumes.find(snapshot.volume)
+      persister.cloud_volume_snapshots.build(
+        :name             => snapshot.name,
+        :size             => cloud_volume.size,
+        :cloud_volume     => cloud_volume,
+        :ems_ref          => snapshot.uuid,
+        :description      => snapshot.description,
+        :status           => snapshot.component_state
```

Comment:
Minior but you don't need this much padding
```suggestion
        :name             => snapshot.name,
        :size         => cloud_volume.size,
        :cloud_volume => cloud_volume,
        :ems_ref      => snapshot.uuid,
        :description  => snapshot.description,
        :status       => snapshot.component_state
```